pulsar-functions/instance/src/main/python/python_instance.py

```diff
@@ -475,6 +509,22 @@ def get_function_status(self):
     status.lastInvocationTime = int(last_invocation) if sys.version_info.major >= 3 else long(last_invocation)
     return status
 
+  def can_enable_effectively_once(self):
+    """
+    The prerequisites for enabling effective-once semantics are as follows.
+    1. deduplication is enabled
+    2. set ProcessingGuarantees to EFFECTIVELY_ONCE
+    3. the function has only one source topic and one sink topic
+    4. (unsupported yet) if partitioned topic is enabled, ensure that the number of partitions
+      (of both source and sink topics) is the same
+    """
+    if self.instance_config.function_details.processingGuarantees == \
+            Function_pb2.ProcessingGuarantees.Value('EFFECTIVELY_ONCE') and \
```

Comment:
Could you please add some log output here when `processingGuarantees` is set to `EFFECTIVELY_ONCE`, but other requirements do not meet?